src/components/ModelingMachineProvider.tsx

```diff
@@ -1727,6 +1758,19 @@ export const ModelingMachineProvider = ({
     }
   )
 
+  // Add debug function to window object
+  useEffect(() => {
+    if (typeof window !== 'undefined') {
+      // @ts-ignore - we're intentionally adding this to window
+      window.getModelingState = () => {
+        return {
+          modelingState,
+          id: modelingState._nodes[modelingState._nodes.length - 1].id,
+        }
+      }
+    }
+  }, [modelingState])
+
```

Comment:
this is just some dedugging to make `getModelingState()` available in the dev console, I find it useful enough, though is unrelated and I can removed it.